src/r5py/r5/travel_time_matrix_computer.py

```diff
@@ -92,10 +100,24 @@ def __init__(
             transport_network = TransportNetwork(*transport_network)
         self.transport_network = transport_network
 
+        # Quick validation to ensure an ID column exists
+        if "id" not in origins.columns:
+            raise NoIDColumnError("Origin dataset must contain an 'id' column.")
+        # And to make sure the column is indeed unique
+        if len(origins.id.unique()) < origins.shape[0]:
```

Comment:
This would be more readable, and probably also better performing: https://pandas.pydata.org/docs/reference/api/pandas.Series.is_unique.html

Reply:
Added a separate issue for this in #162